docs/Modpacks/Materials-and-Elements/TagPrefixes-and-the-power-of-.setIgnored().md

```diff
@@ -1,46 +1,78 @@
 ---
-title: TagPrefixes and the power of .setIgnored()
+title: TagPrefixes and .setIgnored()
 ---
+
+
+# TagPrefixes And The Power Of `.setIgnored()`
+
 ## What is a TagPrefix?
-TagPrefixes are GTCEu Modern's way of streamlining applying item and block tags to Materials, along with some other functions. The `TagPrefix` class, 
-available for use in startup and server scripts, contains a number of predefined TagPrefixes that 
-associate potentially everything from drill heads to flawless gemstones with a material. A common and easy-to-understand example is `TagPrefix.ingot`, 
-which associates with all Materials that have an IngotProperty and thus an associated ingot item, including custom ones defined via KubeJS. 
-TagPrefixes provide localization, item and block tagging, and influence many crafting recipes, and are integral to the functioning of GTCEu's material definition system.
+
+TagPrefixes are GTCEu Modern's way of streamlining applying item and block tags to Materials, along with some other
+functions. The `TagPrefix` class, available for use in startup and server scripts, contains a number of predefined
+TagPrefixes that associate potentially everything from drill heads to flawless gemstones with a material.
+
+A common and easy-to-understand example is `TagPrefix.ingot`, which associates with all Materials that have an
+IngotProperty and thus an associated ingot item, including custom ones defined via KubeJS.  
+TagPrefixes provide localization, item and block tagging, and influence many crafting recipes, and are integral to the
```

Comment:
> tagging, and influence many crafting recipes, and are integral

Redundant and at "and influence", should probably be "[...] tagging, influence many crafting recipes, and are integral [...]"